net-dns/maradns/maradns-3.4.01.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
```

Comment:
What about python3?

Reply:
single script that need it - bind2csv2.py and it not ready to python3.